tool/net/lfuncs.c

```diff
@@ -858,6 +858,73 @@ int LuaUuidV4(lua_State *L) {
   return 1;
 }
 
+int LuaUuidV7(lua_State *L) {
```

Comment:
I like how this code very clearly communicates what's happening. I bet GCC turns it into marvelously small and fast assembly too. You should try comparing it to your previous benchmarks. Do me one favor, and add a comment citing the RFC(s) you're implementing? Also @mrdomino could you give this another glance before we merge it?

Reply:
It performs about the same,  ~half as fast as uuidv4